RecordAutonumber.php

```diff
@@ -168,6 +171,19 @@ public function redcap_every_page_before_render($project_id) {
                         }
 
                 }
+                else if ($this->page==='DataEntry/record_home.php' && isset($_GET['id']) && isset($_GET['auto'])) { 
+
+                        if (isset($_GET['arm']) && array_key_exists($_GET['arm'], $this->Proj->events)) {
+                                $armFirstEventId = key($this->Proj->events[$_GET['arm']]['events']);
+                                $armFirstForm = $this->Proj->eventsForms[$armFirstEventId][0];
+                        } else {
+                                $armFirstEventId = $this->Proj->firstEventId;
+                                $armFirstForm = $this->Proj->firstForm;
+                        }
+                        $tempRecId = $_GET['id'];
+                        $gotoUrl = APP_PATH_WEBROOT."DataEntry/index.php?pid={$this->project_id}&id=$tempRecId&event_id=$armFirstEventId&page=$armFirstForm";
+                        redirect( $gotoUrl);
+                }
```

Comment:
`git-blame` attributes this block to you from 2019-03-07, my merge of your `dev` branch into our `develop` branch did not get rid of it, so I'm pointing it out here so you can decide.